src/components/AddBook.jsx

```diff
@@ -25,20 +25,21 @@ const AddBook = () => {
 
   return (
     <>
-      <h2>ADD NEW BOOK</h2>
-
-      <form onSubmit={sendBook} action="">
-        <input type="text" placeholder="Title" required onChange={(e) => setTitle(e.target.value)} />
-        <input type="text" placeholder="Author" required onChange={(e) => setAuthor(e.target.value)} />
-
-        <select name="categories" id="category" required onChange={(e) => setCategory(e.target.value)}>
-          <option>--Category--</option>
-          <option value="suspense">Suspense</option>
-          <option value="terror">Terror</option>
-          <option value="comedy">Comedy</option>
-        </select>
-        <button type="submit">Add Book</button>
-      </form>
+      <div className="Line container"></div>
```

Comment:
- [ ] I got some errors when I tried to start the app. Please check my screenshot below. I think the issue is coming from the `AddBook.jsx` and `Book.jsx` files.

![a1](https://user-images.githubusercontent.com/39523312/155828471-2c81c731-614a-4f79-838d-ef3b28193bad.png)